styles.css

```diff
@@ -0,0 +1,82 @@
+@import url('https://fonts.googleapis.com/css?family=Poppins:300,400,500,600,700,800,900&display=swap');
+*
+{
+    margin:0;
+    padding:0;
+    box-sizing: border-box;
+    font-family: 'Poppins', sans-serif;
+}
+body{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    min-height: 100vh;
+    background:#091921;
+}
+.header{

```

Comment:
Please add a space after the class name and just before the opening curly brace;
this: `.header {` instead of `.header{`